mne/io/egi/egi.py

```diff
@@ -120,23 +119,29 @@ def read_raw_egi(
         Names of channels or list of indices that should be designated
         MISC channels. Default is None.
     include : None | list
-       The event channels to be ignored when creating the synthetic
-       trigger. Defaults to None.
+       The event channels to be included when creating the synthetic
+       trigger or annotations. Defaults to None.
        Note. Overrides ``exclude`` parameter.
     exclude : None | list
        The event channels to be ignored when creating the synthetic
-       trigger. Defaults to None. If None, channels that have more than
-       one event and the ``sync`` and ``TREV`` channels will be
-       ignored.
+       trigger or annotations. Defaults to None. If None, the ``sync`` and ``TREV``
+       channels will be ignored. This is ignored when ``include`` is not None.
     %(preload)s
 
         .. versionadded:: 0.11
     channel_naming : str
         Channel naming convention for the data channels. Defaults to ``'E%%d'``
         (resulting in channel names ``'E1'``, ``'E2'``, ``'E3'``...). The
         effective default prior to 0.14.0 was ``'EEG %%03d'``.
+        .. versionadded:: 0.14.0
 
-         .. versionadded:: 0.14.0
+    events_as_annotations : bool
```

Comment:
```suggestion
    events_as_annotations : bool | None
```

Reply:
We usually don't add `None` in cases like this where it's just there as a deprecation placeholder